system/jlib/jtrace.cpp

```diff
@@ -449,7 +680,7 @@ class CSpan : public CInterfaceOf<ISpan>
 
             StringBuffer out;
             toLog(out);
-            DBGLOG("Span start: {%s}", out.str());
+            //DBGLOG("Span start: {%s}", out.str());
```

Comment:
at this point the owning thread should fetch the span/trace ids and report them in every existing, related log output